apps/docs/package.json

```diff
@@ -17,25 +17,26 @@
     "@mui/material": "^5.5.1",
     "next": "12.1.0",
     "next-transpile-modules": "^9.0.0",
+    "playwright-config-commons-ui": "workspace:*",
```

Comment:
Should be a dev dependency

Reply:
@kilemensi Eslint complains when i add it to devDependancy

Reply:
Then we'd need to hunt down why and fix that issue @nerdlet. All configs (eslint, jest, storybook, etc.) are dev time dependencies. They shouldn't be packaged with the final build of apps or packages.